velox/type/TimestampConversion.cpp

```diff
@@ -551,6 +554,10 @@ bool tryParseTimestampString(
 
   if (parseMode == TimestampParseMode::kIso8601 && pos < len &&
       buf[pos] == 'T') {
+    if (pos == len - 1) {
+      // If the string is just 'T' with leading spaces.
```

Comment:
nit: shouldnt the leading spaces case be filtered out in  the check in L546-L553.
Did you mean to say that if the input is just 'T'? Can you add a test case for this if not already there?

Reply:
I missed that check, you're right in this case it should just be T (which is still not allowed).  I'll add a test for that.

Reply:
* There already is a test for this, my changes caused it to regress which is why I added this check here.